docs/uninstall-olm.md

```diff
@@ -0,0 +1,62 @@
+# How do I uninstall OLM?
+
+You can uninstall OLM in a symmetrical way as you did to install it. Specifically all OLM resources in [crds.yaml](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/upstream/quickstart/crds.yaml) and [olm.yaml](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/upstream/quickstart/olm.yaml) should be deleted.
+
+## Uninstall Released OLM
+
+For uninstalling release versions of OLM, you can use the following commands:
+
+```bash
+export OLM_RELEASE=<olm-release-version>
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${OLM_RELEASE}/crds.yaml
```

Comment:
curious if you've tested this out? I find that the ordering sometimes doesn't work out in my favor and deleting this way hangs.

Reply:
I found out about that too. If we delete `crds.yaml` first and then `olm.yaml` it would be successful and will not hang. There will be some resources not exist after crds delete, but it will not be a problem. 

```[bowensong@red-bowen ~]$ kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${OLM_RELEASE}/crds.yaml
customresourcedefinition.apiextensions.k8s.io "clusterserviceversions.operators.coreos.com" deleted
customresourcedefinition.apiextensions.k8s.io "installplans.operators.coreos.com" deleted
customresourcedefinition.apiextensions.k8s.io "subscriptions.operators.coreos.com" deleted
customresourcedefinition.apiextensions.k8s.io "catalogsources.operators.coreos.com" deleted
customresourcedefinition.apiextensions.k8s.io "operatorgroups.operators.coreos.com" deleted
[bowensong@red-bowen ~]$ kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${OLM_RELEASE}/olm.yaml
namespace "olm" deleted
namespace "operators" deleted
clusterrole.rbac.authorization.k8s.io "system:controller:operator-lifecycle-manager" deleted
serviceaccount "olm-operator-serviceaccount" deleted
clusterrolebinding.rbac.authorization.k8s.io "olm-operator-binding-olm" deleted
deployment.apps "olm-operator" deleted
deployment.apps "catalog-operator" deleted
clusterrole.rbac.authorization.k8s.io "aggregate-olm-edit" deleted
clusterrole.rbac.authorization.k8s.io "aggregate-olm-view" deleted
Error from server (NotFound): error when deleting "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.13.0/olm.yaml": the server could not find the requested resource (delete operatorgroups.operators.coreos.com global-operators)
Error from server (NotFound): error when deleting "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.13.0/olm.yaml": the server could not find the requested resource (delete operatorgroups.operators.coreos.com olm-operators)
Error from server (NotFound): error when deleting "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.13.0/olm.yaml": the server could not find the requested resource (delete clusterserviceversions.operators.coreos.com packageserver)
Error from server (NotFound): error when deleting "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.13.0/olm.yaml": the server could not find the requested resource (delete catalogsources.operators.coreos.com operatorhubio-catalog)
```

Reply:
I think there's another step of deleting the packageserver apiservice?

Reply:
`Error from server (NotFound): error when deleting "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.13.0/olm.yaml": the server could not find the requested resource (delete clusterserviceversions.operators.coreos.com packageserver)` The packageserver is a CSV kind. When we deleted the CSV as a CRD, all CSVs including `packageserver` are deleted. Therefore, there is no other OLM resource left.